indy_common/types.py

```diff
@@ -334,22 +336,20 @@ class ConstraintEntityField(MessageValidator):
 
 
 class ConstraintListField(MessageValidator):
-    schema = None
+    _base_types = (dict, )
+    schema = ((CONSTRAINT_ID, ChooseField(values=ConstraintsEnum.values())),
+              (AUTH_CONSTRAINTS, IterableField(AnyField())))
 
-    def __init__(self, schema_is_strict=SCHEMA_IS_STRICT):
-        self.schema = (
-            (CONSTRAINT_ID, ChooseField(values=ConstraintsEnum.values())),
-            (AUTH_CONSTRAINTS, IterableField(ConstraintField(self,
-                                                             ConstraintEntityField())))
-        )
-        super().__init__(schema_is_strict)
+    def _validate_message(self, val):
```

Comment:
It looks like we don't do any specific validation for `AUTH_CONSTRAINTS`, so it means that list may contain any values and validation will pass?